src/app/components/form/DualCurrencyField/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { useRef } from "react";
+import { classNames } from "~/app/utils";
+
+export type Props = {
+  suffix?: string;
+  endAdornment?: React.ReactNode;
+  secondaryValue: string | number;
+  label: string;
+};
+
+export default function DualCurrencyField({
+  label,
+  secondaryValue,
+  id,
+  placeholder,
+  type = "text",
+  required = false,
+  pattern,
+  title,
+  onChange,
+  onFocus,
+  onBlur,
+  value,
+  autoFocus = false,
+  autoComplete = "off",
+  disabled,
+  min,
+  max,
+  suffix,
+  endAdornment,
+}: React.InputHTMLAttributes<HTMLInputElement> & Props) {
+  const inputEl = useRef<HTMLInputElement>(null);
+  const outerStyles =
+    "shadow-sm rounded-md border border-gray-300 dark:border-gray-800 bg-white dark:bg-black transition duration-300";
+
+  const inputNode = (
```

Comment:
Wondering if our existing default `Input` can/should be used here

Reply:
@dylancom what do you think?

Reply:
If it's easy to add a second value in (new prop), we should re-use it.

Reply:
https://github.com/getAlby/lightning-browser-extension/issues/1042